packages/sdk/src/document/crdt/tree.ts

```
@@ -1630,6 +1685,25 @@ export class CRDTTree extends CRDTElement implements GCParent {
+    fromLeft: CRDTTreeNode,
+    toParent: CRDTTreeNode,
+    toLeft: CRDTTreeNode,
+    callback: (token: TreeToken<CRDTTreeNode>, ended: boolean) => void,
+  ): void {
+    const fromIdx = this.toIndexIncludeTombstoneNodes(fromParent, fromLeft);
+    const toIdx = this.toIndexIncludeTombstoneNodes(toParent, toLeft);
+    return this.indexTree.tokensBetweenIncludeTombstoneNodes(
+      fromIdx,
+      toIdx,
+      callback,
+    );
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)